docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -27,6 +27,7 @@ The :program:`admin/run-acceptance-tests` script has several options:
 .. option:: --dataset-backend <dataset-backend>
 
    Specifies what dataset backend to use for the cluster.
+   # FLOC-1925 Add a note about available backends.
```

Comment:
I'm not totally happy that we have duplicate CLI parameter documentation at all.  Maybe instead of adding more duplication here, just add a note that `run-acceptance-tests --help` is the canonical source for usage documentation.